applications/nrf5340_audio/src/bluetooth/ble_audio_services.h

```diff
@@ -88,16 +93,59 @@ int ble_vcs_volume_mute(void);
  *         the target device will be itself.
  *
  * @return 0 for success,
- *         -ENXIO if the feature is disabled
+ *         -ENXIO if the feature is disabled.
  *         Other errors from underlying drivers.
  */
 int ble_vcs_volume_unmute(void);
 
 /**
- * @brief Initialize the Volume Control Service server
+ * @brief  Discover MCS and included services.
+ *
+ * @param  conn  Pointer to the active connection, only valid for client.
+ *
+ * @return 0 for success, error otherwise.
+ */
+int ble_mcs_discover(struct bt_conn *conn);
+
+/**
+ * @brief  Get the current state of the media player, only valid for client.
+ *
+ * @param  conn  Pointer to the active connection.
+ *
+ * @return 0 for success, error otherwise.
+ */
+int ble_mcs_state_update(struct bt_conn *conn);
+
+/**
+ * @brief  Send play/pause command to the media player,
+ *         depending on the current state.
+ *
+ * @param  conn  Pointer to the active connection, only valid for client.
+ *               Shall be NULl if called from server.
+ *
+ * @return 0 for success, error otherwise.
+ */
+int ble_mcs_play_pause(struct bt_conn *conn);
+
+/**
+ * @brief Initialize the Volume Control Service server.
  *
  * @return 0 for success, error otherwise.
  */
 int ble_vcs_server_init(void);
 
+/**
+ * @brief Initialize the Media Control Service client.
```

Comment:
Is it correct to call it "Media Control Service server" and "Media Control Service client"?

Reply:
I guess we can call it Media Control Client and Media Control Server